notebooks/notebooks/iclr2022/estimate-flops-adv.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    ) -&gt; float:
</pre><p>Consider adding one-sentence docstring? In particular, it's not obvious what this is actually computing (from context I think it's number of FLOPS *for the game* described by info).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Added comment to clarify.